### PR TITLE
Monitoring cases

### LINKED ISSUE #110: Different R versions can produce different output for return_random_cases() with same seed
**Describe the bug**
Different R versions can produce different output for return_random_cases() with same seed

**To Reproduce**
Run README code on @markomi machine and on @PatrickRWright machine.

**Expected behavior**
Same results.

**Additional context**
https://stackoverflow.com/questions/48626086/same-seed-different-os-different-random-numbers-in-r

https://bugs.r-project.org/bugzilla/show_bug.cgi?id=17494

### PR DESCRIPTION
**Pull request by PatrickRWright**
I added the result of `RNGkind()` to the output for verbosity. Whenever I use this function in future, I will add that to the appendix of the file I produce for the monitoring staff.

I also added the unrelated test to check whether the version number was incremented in the README badge.

closes #110

### CODE REVIEW

**Comment by markomi** [comment]
I guess we should wait with the merge until diff_secuTrial PR has been merged.

**Comment by PatrickRWright** [comment]
Yes, lets stagger the merges according to the version numbers.

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: secuTrialR
 Type: Package
 Title: Handling of data from the clinical data management system secuTrial 
-Version: 0.6.3
+Version: 0.6.6
```
Comment (aghaynes) [inline DESCRIPTION]: what happened to 0.6.5?

File: README.md [hunk 2]
```diff
@@ -451,11 +451,15 @@ return_random_cases(ctu05, percent = 0.25, seed = 1337, date = "2019-03-18",
...
-## 3 RPACK-CBE-003   Charité Berlin (RPACK)      2019-04-03
+## $cases
+##           mnpaid                   centre mnpvisstartdate
+## 1  RPACK-INS-011 Inselspital Bern (RPACK)      2019-04-11
+## 4  RPACK-INS-014 Inselspital Bern (RPACK)      2019-04-14
+## 41 RPACK-CBE-004   Charité Berlin (RPACK)      2019-04-04
+## 3  RPACK-CBE-003   Charité Berlin (RPACK)      2019-04-03
+## 
+## $rng_config
+## [1] "Mersenne-Twister" "Inversion"
```
Comment (aghaynes) [inline README.md]: More of a note than anything else... you don't have to commit the whole file... you can select just the badge to commit, then this part wont change all the time

File: DESCRIPTION (hunk 1 above)
Comment (PatrickRWright) [reply DESCRIPTION]: @markomi  is working on something which will be 0.6.5

File: README.md (hunk 2 above)
Comment (PatrickRWright) [reply README.md]: Would you do that manually or is there a way to commit specific lines?

Comment (aghaynes) [reply README.md]: in rstudio you can select which lines to stage. This SO post has a method for the command line https://stackoverflow.com/questions/1085162/commit-only-part-of-a-file-in-git

**Comment by markomi** [comment]
Thanks Patrick!

**Pull request merged**